x-pack/legacy/plugins/siem/public/pages/detection_engine/components/signals_chart/index.tsx

```diff
@@ -36,4 +36,8 @@ export const SignalsCharts = memo(() => (
 
     <HistogramSignals />
   </EuiPanel>
-));
+);
+
+SignalsChartsComponent.displayName = 'SignalsChartsComponent';
+
+export const SignalsCharts = memo(SignalsChartsComponent);
```

Comment:
Same question here for why we're abstracting the `memo` to a parent component? (This component has no props either fwiw)